File: InClassCode/Week2Code/InClassLibraryExample/book.py
class Book:
    def __init__(self, title, author):
        self._title = title
        self._author = author
        self._checked_out = False

    def get_title(self):
        return self._title

    def get_author(self):
        return self._author

    def get_checked_out(self):
        return self._checked_out

    def change_status(self):
        self._checked_out = not self._checked_out

File: InClassCode/Week2Code/InClassLibraryExample/Book.java
public class Book {
    private String title;
    private String author;
    private boolean checkedOut;

    public Book(String title, String author) {
        this.title = title;
        this.author = author;
        this.checkedOut = false;
    }
    public String getTitle() {
        return this.title;
    }
    public String getAuthor() {
        return this.author;
    }
    public boolean getCheckedOut() {
        return this.checkedOut;
    }

    public void changeStatus() {
        this.checkedOut = !this.checkedOut;
    }
}
